lib/containers/engine.pm

```diff
@@ -109,9 +109,23 @@ sub build {
     my ($self, $dockerfile_path, $container_tag, %args) = @_;
     die 'wrong number of arguments' if @_ < 3;
 
-    #TODO add build with URL https://docs.docker.com/engine/reference/commandline/build/
-    $self->_engine_assert_script_run("build -f $dockerfile_path/Dockerfile -t $container_tag $dockerfile_path", $args{timeout} // 300);
```

Comment:
What about `script_retry`?

Reply:
I think we need to do a cleanup of any possible intermediate image layer that have been created.

Reply:
ok, will try that, thanks for the suggestion :)
But I'll wait until https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14452 is merged and rebase on top.